graylog2-server/src/main/java/org/graylog/events/conditions/Expr.java

```diff
@@ -314,4 +316,32 @@ public Double accept(ExpressionVisitor visitor) {
             return visitor.visit(this);
         }
     }
+
+    @AutoValue
+    public static abstract class Group implements Expression<Boolean> {
+        static final String EXPR = "group";
+
+        @JsonProperty(CHILD)
+        public abstract Expression<Boolean> child();
+
+        @JsonProperty(OPERATOR)
```

Comment:
maybe we could add a comment, that the operator isn't actually used for evaluating expressions?